src/main/java/com/k3ntako/HTTPServer/controllers/Reminders.java

```diff
@@ -71,4 +71,16 @@ public Response put(RequestInterface request) throws HTTPError {
 
     return new Response();
   }
+
+  public Response delete(RequestInterface request) throws HTTPError {
+    var id = request.getParam("id");
+
+    try {
+      textFile.delete(id);
+    } catch (IOException e) {
+      throw new HTTPError(404, "Reminder was not found");
+    }
```

Comment:
I only catch `IOException`s, but should I make sure that the message is the specific `IOException` that I am throwing in `FileIO` on line 64?